awswrangler/athena/_write_iceberg.py

```diff
@@ -86,6 +86,7 @@ def to_iceberg(
     boto3_session: Optional[boto3.Session] = None,
     s3_additional_kwargs: Optional[Dict[str, Any]] = None,
     additional_table_properties: Optional[Dict[str, Any]] = None,
+    dtype: Optional[Dict[str, str]] = None,
```

Comment:
The `dtype` argument must also be handled in the `_create_iceberg_table` function below (i.e. when creating an Iceberg table).

Could you also please add a unit test to check that we have not missed anything else besides the above?

Reply:
That's right, I have forgot about the scenario of new tables... I believe that it should be adjusted now...

I have also tried to add an unit test similar to the `s3.to_parquet`, I hope it works...